index.html

```diff
@@ -72,9 +72,9 @@ <h2><a href="#">Nemwel Boniface</a></h2>
 
         <div class="navLinks">
           <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#aboutme">About</a></li>
+            <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Great job with the menu, but as per the requirement screenshotted below the menu should disappear, while it changes to the correct part of the page, the menu is still up there over the "authordetails" section, kindly make it in a way that the menu closes with similar behavior to pressing the x button
![image](https://user-images.githubusercontent.com/73140461/150833926-9948e54c-0aee-4f45-ab3f-c4a6b05e82fd.png)

Reply:
Hello @leolpaz thank you for this review. I have managed to make the menu bar disappear when you click the link. I have used a for..of loop and I am getting this error when running my linters. Any ideas of help please ? 
"iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations"

Reply:
Can you try using forEach on lists instead of the type of loop you're using?

Reply:
Hello @leolpaz  I tried using forEach and it worked! thank you